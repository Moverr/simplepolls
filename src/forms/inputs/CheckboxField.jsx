import React from 'react'
import InputField from './InputField';



function CheckboxField(props){
    return <InputField type="checkbox" {...props} />; 
}
export default CheckboxField;