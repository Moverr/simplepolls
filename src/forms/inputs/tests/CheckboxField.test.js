import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
 
import CheckboxField from '../CheckboxField';
 

describe('Testing Input FIeld', () => {
  it('should  render correctly', () => {    
    const props = {
        className:"inputfield",
        value:"Label Title",
         name:"name" 
    }

      const wrapper = shallow(<CheckboxField props />);
      expect(wrapper).toMatchSnapshot();
  });
})



 