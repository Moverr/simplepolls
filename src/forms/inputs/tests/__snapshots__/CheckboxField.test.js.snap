// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Input FIeld should  render correctly 1`] = `
<InputField
  props={true}
  type="checkbox"
/>
`;
